app/airq/commands/donate.py

```diff
@@ -0,0 +1,17 @@
+from flask_babel import gettext
+
+from airq.commands.base import MessageResponse
+from airq.commands.base import RegexCommand
+from airq.models.events import EventType
+
+
+class ShowAbout(RegexCommand):
+    pattern = r"^7[\.\)]?$"
+
+    def handle(self) -> MessageResponse:
+        self.client.log_event(EventType.DONATE)
+        return MessageResponse(
```

Comment:
We are going to need translations for this and the other strings that are modified or added in this PR. @emjgreen what is the website we're using for this again?